package.json

```diff
@@ -196,6 +197,7 @@
     "@storybook/addon-storyshots": "^5.3.17",
     "@storybook/addons": "^5.3.17",
     "@storybook/react": "^5.3.17",
+    "@svgr/cli": "^5.4.0",
     "@svgr/rollup": "^5.4.0",
```

Comment:
We can remove the `@svgr/rollup` dep and refactor the rollup config to no longer do that step, correct?

Reply:
I believe so. I'll test it now

Reply:
Removed